src/components/NavBar/MenuDropdown.tsx

```diff
@@ -140,24 +146,37 @@ export const MenuDropdown = () => {
                 <Box display={{ sm: 'none', lg: 'flex', xxl: 'none' }}>
                   <PrimaryMenuRow to="/pool" close={toggleOpen}>
                     <Icon>
-                      <PoolIcon width={24} height={24} color={theme.textSecondary} />
+                      <PoolIcon width={24} height={24} fill={theme.textPrimary} />
                     </Icon>
                     <PrimaryMenuRow.Text>
                       <Trans>Pool</Trans>
                     </PrimaryMenuRow.Text>
                   </PrimaryMenuRow>
                 </Box>
+                <Box
```

Comment:
Is there a reason this uses a custom `<Box>` instead of the `<DownloadButton>`?

Reply:
I'm following this file's convention of using the display property of Box for responsiveness. <DownloadButton /> is visually very different than this inline overflow menu item.